curriculum/challenges/english/25-front-end-development/workshop-fruit-search-app/67ecfbfe17c7d24465e0de88.md

```diff
@@ -0,0 +1,125 @@
+---
+id: 67ecfbfe17c7d24465e0de88
+title: Step 4
+challengeType: 0
+dashedName: step-4
+---
+
+# --description--
+
+Inside the `form` element, add a search input with an `id` of `search-input` and `type` `search`.
```

Comment:
we can't have two words in backticks next to each other.
https://github.com/freeCodeCamp/freeCodeCamp/issues/60039

that needs to be fixed